caas/kubernetes/provider/k8s.go

```diff
@@ -192,7 +192,9 @@ func (k *kubernetesClient) PrepareForBootstrap(ctx environs.BootstrapContext) er
 
 // Create implements environs.BootstrapEnviron.
 func (k *kubernetesClient) Create(context.ProviderCallContext, environs.CreateParams) error {
-	return nil
+	// must raise errors if it's already exist.
```

Comment:
Now that this method is filled in, there should be a small unit test for it

Reply:
done.